app/uk/gov/hmrc/bankaccountverificationexamplefrontend/example/PetDetails.scala.html

```diff
@@ -98,6 +110,57 @@ <h1 class="govuk-heading-xl">@messages("petDetailsPage.heading")</h1>
             name = detailsForm("description").name, id = detailsForm("description").name,
             errorMessage = detailsForm("description").error.map(fe ⇒ ErrorMessage(content = HtmlContent(messages(fe.message))))))
 
+        <br/>
+        <br/>
+        <h2 class="govuk-heading-m govuk-hint">(Optional) Enter data to pre-populate</h2>
```

Comment:
<an `<h2>` shouldn't be nested in a `<p>`

Reply:
Not sure where the unclosed <p> tag is?

Reply:
just noticed the line numbers, there is a big gap...

Ignore me on that one.